chainerx_cc/chainerx/routines/math.cc

```diff
@@ -599,6 +637,14 @@ Array Minimum(const Array& x1, Scalar x2) {
 
 Array Minimum(Scalar x1, const Array& x2) { return Minimum(x2, x1); }
 
+Array Minimum(const Array& x1, const Array& x2) {
+    Dtype dtype = GetArithmeticResultDtype(x1, x2);
+    if (GetKind(dtype) != DtypeKind::kFloat) {
+        dtype = internal::GetDefaultDtype(DtypeKind::kFloat);
```

Comment:
This function seems better to return integers when integer arrays are given as the arguments, as NumPy's one does so.